src/main/java/uk/gov/hmcts/reform/managecase/domain/notify/EmailNotificationRequestSuccess.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.managecase.domain.notify;
+
+import uk.gov.service.notify.SendEmailResponse;
+
+public class EmailNotificationRequestSuccess extends EmailNotificationRequestStatus {
+
+    private final SendEmailResponse emailResponse;
+
+    public EmailNotificationRequestSuccess(EmailNotificationRequest notificationRequest,
+                                           SendEmailResponse emailResponse) {
+        super(notificationRequest);
+        this.emailResponse = emailResponse;
+    }
+
+    @Override
+    public Object getNotificationStatus() {
+        return this.emailResponse;
+    }
+}
```

Comment:
`public SendEmailResponse getResponse() {
        return this.emailResponse;
    }`